src/WindowManager.cpp

```diff
@@ -107,10 +108,52 @@ void WindowManager::checkForUserChanges() {
         mWasPressed = false;
     }
 
-    if (isKeyPressed(GLFW_KEY_ESCAPE)) {
```

Comment:
I'll do. But we need up and down and then exiting the menu too. I'm done with the menu now (just need to change one of the textures), so maybe we can merge and you'll add the controller stuff?